org.kde.kontact.json

```diff
@@ -0,0 +1,1224 @@
+                        },
+                        {
+                            "name": "xsd",
+                            "make-install-args": [
+                                "install_prefix=/app"
+                            ],
+                            "no-autogen": true,
+                            "sources": [
+                                {
+                                    "type": "archive",
+                                    "url": "http://www.codesynthesis.com/download/xsd/4.0/xsd-4.0.0+dep.tar.bz2",
+                                    "sha256": "eca52a9c8f52cdbe2ae4e364e4a909503493a0d51ea388fc6c9734565a859817"
+                                }
+                            ],
+                            "cleanup": [
+                                "/share/doc/xsd",
+                                "/share/doc/libxsd"
+                            ]
+                        },
+                        {
+                            "name": "kdelibs4support",
+                            "config-opts": [
+                                "-DBUILD_TESTING=OFF",
+                                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+                            ],
+                            "buildsystem": "cmake-ninja",
+                            "sources": [
+                                {
+                                    "type": "archive",
+                                    "url": "https://download.kde.org/stable/frameworks/5.50/portingAids/kdelibs4support-5.50.0.tar.xz",
+                                    "sha256": "b4800719c22fbb6643ea17c83eb01fe4468604c19c5bfd89430dba6b59b8348a"
+                                }
+                            ]
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "ktnef",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/applications/18.08.1/src/ktnef-18.08.1.tar.xz",
+                    "sha256": "e2c3238f5d500b2f67b1ad2b91cd480650d8004e514efa69e6551596f2d391a0"
+                }
+            ]
+        },
+        {
+            "name": "kmail",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/applications/18.08.1/src/kmail-18.08.1.tar.xz",
+                    "sha256": "747eac028620da6c6aaf9b00d0d6ab3eabea23dcbad294b26ebff4c6ac3c0988"
+                }
+            ]
+        },
+        {
+            "name": "akregator",
```

Comment:
This build appears to fail on ARM, nothing terribly obvious why - https://flathub.org/builds/#/builders/48/builds/415

Reply:
Looks like a bug in Qt moc - the generated code refers to a class in a wrong namespace. I added a patch that tries to solve it by moving the class declaration to the namespace where moc expects it, let's see if it helps :)